hil/client/README.md

```diff
@@ -0,0 +1,24 @@
+### HIL Python Client API
```

Comment:
* Start with level 1 headings (one `#`)
* This should go under docs somewhere.
* Probably should call it "client library"
* This could use some introductory verbiage; You're kindof just plopping some code down without any overall context. Also, we don't describe the design of the library at all; it would be good to briefly describe the important objects and sketch its structure (docstrings can serve as comprehensive reference docs, but we should give an overview).

Reply:
> This should go under docs somewhere.
Personally, I don't like putting this under `/docs` since there are too many docs there already and people probably won't read it. So I put it in the client directory. But if it is the best practice then I will move it there. Maybe creating a new folder called `/client`?

Point 1 & 3 sound good to me.

As for the point 4, I thought we have the overview description when the client library first got introduced to the code base. Will discuss with @naved001 and try to come up with an overview. The idea sounds good to me, but I don't have a plan on how to write one atm.

Reply:
Re: `/docs`, this is where all of our high-level docs have gone. Maybe we should update the README at the root of the repo to provide a more thorough index, if we want stuff to be easier to find.

I'm looking at the client library as something that people who don't hack on HIL itself might hypothetically use; having it buried inside of the source code doesn't make much sense. It might be more reasonable if this were an internal-use-only module.

Re (4): if we have overview docs, I don't know where they are.